PusherClient/EventEmitter.cs

```diff
@@ -28,6 +28,26 @@ public void BindAll(Action<string, dynamic> listener)
             _generalListeners.Add(listener);
         }
 
+        public void Unbind(string eventName)
+        {
+            _eventListeners.Remove(eventName);
+        }
+
+        public void Unbind(string eventName, Action<dynamic> listener)
+        {
+            if(!_eventListeners.ContainsKey(eventName))

```

Comment:
ahh, good shout. I think when I wrote the if...return I was anticipating it being more than one line 😆